src/Controller/UI/DatalandController.php

```diff
@@ -101,7 +101,7 @@ public function defaultAction(string $udi)
                 ->andwhere('a.subjectEntityName = ?2')
                 ->andwhere('a.actionName = ?3')
                 ->setParameter(1, $dataset->getId())
-                ->setParameter(2, 'App\Entity\Dataset')
+                ->setParameter(2, 'Pelagos\Entity\Dataset')
```

Comment:
All this logic should be taken out, and dropped into a utility or something.
Also maybe we can resolve this by only using the main entity name like: "Dataset".